assets/src/customizer/amp-customize-controls-legacy.js

```diff
@@ -0,0 +1,347 @@
+/* global _, jQuery */
+
+/* eslint no-magic-numbers: [ "error", { "ignore": [ 0, 1, 250] } ] */
+
+window.ampCustomizeControls = ( function( api, $ ) {
```

Comment:
This file is simply renamed from `amp-customize-controls.js`. No changes have been made otherwise.